src/mmrelay/plugins/mesh_relay_plugin.py

```
@@ -211,16 +262,38 @@ async def handle_room_message(self, room, event, full_message) -> bool:
 
-        meshtastic_client._sendPacket(
-            meshPacket=meshPacket, destinationId=packet["toId"]
-        )
+        # _sendPacket is required for relaying raw MeshPacket payloads.
+        # Note: this is a private API; monitor upstream Meshtastic changes.
+        try:
+            meshtastic_client._sendPacket(meshPacket=meshPacket, destinationId=to_id)
+        except AttributeError:
+            self.logger.error(
+                "_sendPacket method not available; Meshtastic API may have changed"
+            )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)